openapi-build-extension/src/main/java/io/ballerina/openapi/build/HttpServiceAnalysisTask.java

```diff
@@ -80,11 +86,19 @@ public void perform(SyntaxNodeAnalysisContext context) {
         if (!buildOptions.exportOpenAPI()) {
             return;
         }
+        // if there are any compilation errors, do not proceed
+        if (!isErrorPrinted) {
+            setIsErrorPrinted(true);
+            PrintStream outStream = System.out;
+            outStream.println("openapi contract generation is skipped because the compilation error(s)" +
+                    " in the ballerina package");
```

Comment:
```suggestion
            outStream.println("openapi contract generation is skipped because of the compilation error(s)" +
                    " in the ballerina package");
```